content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -34,7 +34,10 @@ The following fields are returned if valued for clinical documents:
 * [Indexed date/time](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.indexed){:target="_blank"}
 * [Status (typically current)](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.status){:target="_blank"}
 * [Document status (typically final or amended)](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.docStatus){:target="_blank"}
-* [ContentType and URL (fully qualified link to the document](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
+* [Document Attachment](https://hl7.org/fhir/dstu2/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
```

Comment:
i'd prefer not to include a blank line

Reply:
[removed blank line](https://github.com/cerner/fhir.cerner.com/pull/337/commits/51acc559d2f17d86dad80ac82201628a4caaf31b#diff-b5f0beeb0c76f04f4fcbe824eb832abeL43)